src/ImportExport/.gitignore

```diff
@@ -0,0 +1,16 @@
+bin
+obj
+.vs
+generated
+internal
+exports
+tools
+custom/*.psm1
+custom/autogen-model-cmdlets
+test/*-TestResults.xml
+/*.ps1
+/*.ps1xml
+/*.psm1
+/*.snk
+/*.csproj
+/*.nuspec
```

Comment:
Should we add git attributes and gitignore file here?

Reply:
This is automatically generated by autorest powershell.